### PR TITLE
add Comment entity

### PR DESCRIPTION
**Pull request by songhee24**

### CODE REVIEW

File: src/main/java/peaksoft/entity/Comment.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+@Getter
+@Setter
+@NoArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class Comment {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "comment_id")
+    Long commentId;
```
Comment (kanatsubanov321) [inline src/main/java/peaksoft/entity/Comment.java:22]: make it id not commentId

File: src/main/java/peaksoft/entity/Comment.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "Comment")
```
Comment (kanatsubanov321) [inline src/main/java/peaksoft/entity/Comment.java:12]: it's not needed

File: src/main/java/peaksoft/entity/Comment.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+@Getter
+@Setter
```
Comment (kanatsubanov321) [inline src/main/java/peaksoft/entity/Comment.java:14]: you can use one annotation for getters and setters

File: src/main/java/peaksoft/entity/Comment.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class Comment {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "comment_id")
+    Long commentId;
+
+    @Column(name = "text")
+    String text;
```
Comment (kanatsubanov321) [inline src/main/java/peaksoft/entity/Comment.java:25]: how do you think? do we need to add text size?

Comment (songhee24) [reply src/main/java/peaksoft/entity/Comment.java:25]: yes we do)

**Pull request closed without merging**